server/controllers/index.js

```diff
@@ -61,13 +65,15 @@ export const series = async(ctx, next) => {
     total,
     items
   };
+  const pagination = getSeriesPagination(series, parseInt(page, 10) || 1);
 
   ctx.render('pages/list', {
     pageConfig: createPageConfig({
       title: name,
       inSection: 'explore'
     }),
-    list: series
+    list: series,
```

Comment:
being really picky here, you could just rename `series` to `list` and just do `list,` as you already know it's a `Series` from the type annotation